doc-content/enterprise-only/installation/assembly_installing-on-jws-using-installer.adoc

```diff
@@ -1,7 +1,7 @@
 [id='assembly_installing-using-installer']
-= Using the installer to install {PRODUCT} on {EAP} or Red Hat JBoss Web Server 
+= Using the installer to install {PRODUCT} on {EAP} or {JWS} 
 
-This section describes the steps required to install {PRODUCT} using the installer JAR file. The JAR file is an executable file that installs {PRODUCT} in an existing {EAP} 7.1 or Red Hat JBoss Web Server 5.0 server installation. You can run the installer in standard or command line interface (CLI) mode.
+This section describes the steps required to install {PRODUCT} using the installer JAR file. The JAR file is an executable file that installs {PRODUCT} in an existing {EAP} 7.1 or {JWS} {JWS_VERSION_LONG} server installation. You can run the installer in standard or command line interface (CLI) mode.
```

Comment:
`7.1` should be replaced with EAP_VERSION

Reply:
Please ignore EAP  references - those are updated in another fork. I know it's a little confusing but because we single source some files, you see seemingly outdated info for  other platforms. You can confirm this in the rendered drafts, @domhanak.